### ISSUE TITLE
Create and deploy DIBBs storage account for state management

### ISSUE THREAD

**Issue by rin-skylight** [CDCgov/dibbs-cloud#38]
### Background
We can't begin working with TF until we have our storage account and state elements ready to go. We need this before we begin work on rolling out the other elements of the DIBBs cluster.

### Action requested
Create the storage account. This can be done manually or via a single-use TF script (see phdi-playground for this). The info then needs to be populated in any relevant TF scripts that already exist.

### Acceptance Criteria
- Storage account exists
- Any TF scripts or GitHub variables that need to be updated are, in fact, properly updated.

**Comment by rin-skylight** [comment]
Duplicate of #35.
